projects/plugins/jetpack/extensions/blocks/recurring-payments/edit.jsx

```diff
@@ -15,17 +15,40 @@ import { getBlockStyles } from './util';
 const BLOCK_NAME = 'recurring-payments';
 
 export default function Edit( { attributes, clientId, setAttributes } ) {
-	const { align, planId, width } = attributes;
-	const planIds = useMemo( () => {
-		const _planIds = ( '' + planId ).split( '+' ).map( id => parseInt( id, 10 ) );
-		return _planIds;
-	}, [ planId ] );
+	const { align, planId, planIds, width } = attributes;
+
+	// planId is a integer, planIds is an array.
+	// if planIds is set, use it, otherwise use planId. Going forward we should only use planIds.
+	const _planIds = useMemo( () => {
```

Comment:
This code isn't expensive enough to be in a `useMemo`

Reply:
Same response as above (do you have a better solution to the linter?)

Reply:
That makes sense. Might want to add a comment that these are so that the resolved value can be used in other hooks.